### PR TITLE
Added gaussian filter to preprocess script

### PR DESCRIPTION
**Pull request by AlonDaks**

### CODE REVIEW

File: code/stat159lambda/preprocess/preprocess.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+def concatenate_runs(subj_num):
+    npy_file_name = dp.get_concatenated_path(subj_num)
+    if not exists(npy_file_name) or not USE_CACHED_DATA:
+        run_data = []
+        for j in range(NUM_RUNS):
+            img = nib.load(dp.get_raw_path(subj_num, j + 1))
+            data = img.get_data()
+            if j == 0:
+                run_data.append(data[..., :-4])
+            elif j >= 1 and j <= 6:
```
Comment (jodreen) [inline code/stat159lambda/preprocess/preprocess.py]: should clarify what 1 and 6 mean

File: code/stat159lambda/preprocess/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+
+def merge_2d_data(data_slices):
+    return np.hstack(tuple(data_slices))
+
+
+def get_affine():
+    affine_path = '{0}/data/affine.npy'.format(REPO_HOME_PATH)
+    if not exists(affine_path):
+        img = nib.load(dp.get_raw_path(1, 1))
```
Comment (jodreen) [inline code/stat159lambda/preprocess/preprocess.py]: clarify what 1, 1 means

**Comment by jodreen** [comment]
i think documentation would be helpful, not exactly sure where you're getting a lot of constants, and what each function requires

File: code/stat159lambda/preprocess/preprocess.py (hunk 1 above)
Comment (AlonDaks) [inline code/stat159lambda/preprocess/preprocess.py]: You're right -- it rephrased 6 in terms of NUM_RUNS - 2

File: code/stat159lambda/preprocess/preprocess.py (hunk 2 above)
Comment (AlonDaks) [inline code/stat159lambda/preprocess/preprocess.py]: done

**Comment by AlonDaks** [comment]
I will add documentation in the next PR -- working to close this so I can begin testing on Amazon EC2

**Comment by lisaannyu** [comment]
Ok I'll approve this.  Just to be sure to add documentation for the next PR

**Pull request merged**